buildozer/default.spec

```
@@ -192,7 +192,7 @@ fullscreen = 0
 # (bool) Enable AndroidX support. Enable when 'android.gradle_dependencies'
 # contains an 'androidx' package, or any package from Kotlin source.
 # android.enable_androidx requires android.api >= 28
-#android.enable_androidx = False
+android.enable_androidx = True
```

How about leaving the option commented (but with the updated default):
```suggestion
#android.enable_androidx = True
```

and having the default changed here?

https://github.com/kivy/buildozer/blob/b8575fd5b1e150bf93be3b7e77f229a28a7e5e0d/buildozer/targets/android.py#L1265-L1267

That will allow us to enable it by default, even for current users that may have an outdated `buildozer.spec`